src/main.py

```diff
@@ -0,0 +1,34 @@
+import click
+import yaml
+
+from langchain_huggingface import HuggingFaceEmbeddings
+from src.pdf_to_text.extraction import get_sentences_from_pdf
+from src.llm.bielik import LLM
+from vector_db.vector_db import create_vector_db
+
+
+@click.command()
+@click.option('--config', '-c', default='../config.yaml', help='Path to the configuration file')
+def run_pipeline(config) -> None:
+    with open(config, 'r') as file:
+        config_data = yaml.safe_load(file)
+
+        pdf_path = config_data.get('pdf_path')  # load path to pdf

```

Comment:
Maybe we will use pydantic model here to load data from config, but I have to think about it. 
For the reference: 
https://docs.pydantic.dev/latest/concepts/pydantic_settings/#usage